src/app/navbar.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import styles from "./Navbar.module.css";
+
+export default function Navbar() {
+  const pathname = usePathname();
+
+  const pageLinksData = [
+    {
+      name: "Home",
+      url: "/",
+    },
+    {
+      name: "About",
+      url: "/about",
+    },
+  ];
+
+  const socialLinksData = [
+    {
+      name: "Discord",
+      url: "https://discord.gg/5SyB3yx",
+      icon: "/discord_mark.svg",
+    },
+    {
+      name: "GitHub",
+      url: "https://github.com/ufssd",
+      icon: "/github_mark.svg",
+    },
+    {
+      name: "LinkedIn",
+      url: "https://www.linkedin.com/company/ssduf",
+      icon: "/linkedin_logo.png",
+    },
+    {
+      name: "YouTube",
+      url: "https://www.youtube.com/@ufssd",
+      icon: "/youtube_icon.png",
+    },
+    {
+      name: "Instagram",
+      url: "https://www.instagram.com/uf.ssd/",
+      icon: "/instagram_glyph.svg",
+    },
+  ];
+
+  const pageLinks = pageLinksData.map((link) => (
+    <Link
+      key={link.name}
+      href={link.url}
+      className={pathname === link.url ? styles.pageLinkOpen : ""}
+    >
+      {link.name}
+    </Link>
+  ));
+
+  const socialLinks = socialLinksData.map((link) => (
+    <a
+      key={link.name}
+      target="_blank"
+      rel="noopener noreferrer"
+      href={link.url}
+    >
+      <Image src={link.icon} alt={`${link.name} Logo`} width="32" height="32" />
```

Comment:
[Question]: If `width`/`height` are set here, why is it autoscaling the image for mobile views?

Reply:
In the file `.next/static/css/app/layout.css` exists the following selector:

```css
/*
Constrain images and videos to the parent width and preserve their intrinsic aspect ratio. (https://github.com/mozdevs/cssremedy/issues/14)
*/

img,
video {
  max-width: 100%;
  height: auto;
}
```

This, as stated in the comment, causes the `img` elements by default to shrink when the container size shrinks.